L1Trigger/TrackTrigger/src/StubPtConsistency.cc

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
+#include <memory>
+
+#include "L1Trigger/TrackTrigger/interface/StubPtConsistency.h"
+
+namespace StubPtConsistency {
+
+  float getConsistency(TTTrack< Ref_Phase2TrackerDigi_ > aTrack, const TrackerGeometry* theTrackerGeom, const TrackerTopology* tTopo, double mMagneticFieldStrength, int nPar) {
+
+    if( !(nPar==4 || nPar==5) ) throw cms::Exception("IncorrectInput") << "Not a valid nPar option!";
+
+    double trk_bendchi2 = 0.0;
+    double bend_resolution = 0.483;
+    float speedOfLight = 3.0E8; //in m/s
```

Comment:
we recommend using `CLHEP::c_light` for consistency throughout CMSSW. Examples: https://github.com/cms-sw/cmssw/search?q=clhep+c_light&unscoped_q=clhep+c_light

Also, if you will always use `speedOfLight/UnitConversion` together, you should define a single constant for those and use it everywhere, in order to do the division only once.